Loop/Managers/WatchDataManager.swift

```diff
@@ -194,7 +194,7 @@ final class WatchDataManager: NSObject {
             
             if let glucose = glucose {
                 updateGroup.enter()
-                manager.glucoseStore.getCachedGlucoseSamples(start: glucose.startDate) { (samples) in
+                manager.glucoseStore.getCachedGlucoseSamples(start: glucose.startDate, end: nil) { (samples) in
```

Comment:
I had to add all of these `end: nil`s because protocols don't have default arguments